tests/opaque.test.js

```diff
@@ -774,3 +777,25 @@ describe("server.finishLogin", () => {
     );
   });
 });
+
+describe("server.getPublicKey", () => {
```

Comment:
can you add a test to check that the publicKey from this function is the same one as in `ClientLoginFinishResult` and `ClientRegistrationFinishResult`

Reference: https://docs.rs/opaque-ke/latest/opaque_ke/#checking-server-consistency

Reply:
there is a check for this in the full registration and login flow test

Reply:
ahhh nice, missed that, thx!